### DISCUSSION TITLE
v3.1.29

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#10515] [General]
## What's Changed
* Update composer.json to use Larastan Org by @arnebr in https://github.com/filamentphp/filament/pull/10489
* Update Vietnamese translations by @datlechin in https://github.com/filamentphp/filament/pull/10493
* Update 04-pages.md by @mwaqasiu in https://github.com/filamentphp/filament/pull/10499
* Use wire:navigate when redirecting by @PhiloNL in https://github.com/filamentphp/filament/pull/10477
* fix: Font names with spaces by @danharrin in https://github.com/filamentphp/filament/pull/10503
* fix: Query builder active count by @danharrin in https://github.com/filamentphp/filament/pull/10507
* feature: modifyFormFieldUsing() by @danharrin in https://github.com/filamentphp/filament/pull/10509
* fix: Table nested action arguments by @danharrin in https://github.com/filamentphp/filament/pull/10510
* Fixed typos in European Translation by @Corvisier in https://github.com/filamentphp/filament/pull/10514

## New Contributors
* @arnebr made their first contribution in https://github.com/filamentphp/filament/pull/10489
* @mwaqasiu made their first contribution in https://github.com/filamentphp/filament/pull/10499

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.1.28...v3.1.29

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.1.29'>v3.1.29</a>.</em>

**Comment by IgordeOliveira** [comment]
It's nice to see the pace of updates of you guys 🙏
